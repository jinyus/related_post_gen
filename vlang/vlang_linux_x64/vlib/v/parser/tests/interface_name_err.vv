interface A {
	a int
	b() int
}

struct Ba {
	a int
}

fn (b &Ba) b() int {
	return b.a
}

fn main() {
	a := &Ba{
		a: 1
	}
	b := A(a)
}
