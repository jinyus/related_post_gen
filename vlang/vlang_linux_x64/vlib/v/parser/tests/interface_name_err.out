vlib/v/parser/tests/interface_name_err.vv:1:11: error: single letter capital names are reserved for generic template types.
    1 | interface A {
      |           ^
    2 |     a int
    3 |     b() int
